test/deps/aws_iam_mysql_server/Makefile

```
@@ -0,0 +1,22 @@
+include $(PROXYSQL_PATH)/include/makefiles_vars.mk
+include $(PROXYSQL_PATH)/include/makefiles_paths.mk
+
+.DEFAULT_GOAL := all
+
+CPPFLAGS := -I$(SSL_IDIR)
+CXXFLAGS := $(STDCPP) -O0 -ggdb -Wall -Wextra -Werror $(WASAN)
+LDFLAGS := -L$(SSL_LDIR) $(WASAN)
+LDLIBS := -lssl -lcrypto -lpthread
+
+.PHONY: all clean
+all: aws_iam_mysql_server-t
```

checkmake (0.3.2)

[warning] Required target "test" is missing from the Makefile.

(minphony)